refTemplates/03_Agent_Communication/mcp_agent_mail_rust/README.md

```
@@ -0,0 +1,6 @@
+Skeleton for mcp_agent_mail_rust
```

markdownlint-cli2 (0.23.2)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)